Cargo.toml

```diff
@@ -33,7 +33,7 @@ postgres-native-tls = "0.5.0"
 native-tls = "0.2"
 x509-cert = { version = "0.2.5", features = ["pem"] }
 serde_path_to_error = "0.1.2"
-octocrab = "0.30.1"
+octocrab = { version = "0.30.1", features = ["stream"] }
```

Comment:
We should about updating the `octocrab` sometime in the future, our version seems quite old.